src/content/docs/style-guide/writer-workflow/github-troubleshooting.mdx

```diff
@@ -102,16 +102,20 @@ If you're running with issues with your local build, try these options:
 Redirects are a bit strange on local builds. To test them, navigate to the page that is being redirected, wait until it throws a 404, and then wait ~1-2 minutes. It should redirect you after a while. If it doesn’t, ensure you set up the redirect correctly. 
 
 ## A check fails in the PR [#check-fail]
+<Callout variant='important'>
+The only checks needed to merge a PR are the checks marked as `required` on the PR. These are `run linter` and `run tests` for merges to develop, and `build the docs site` for merges to main. If a required check fails, the failure must be addressed in order for the PR to be merged. If an optional check fails, reach out to the help channel hero so that they can look into the failure, but feel free to merge the PR since optional checks don't block releases.
```

Comment:
just to clarify, maybe we should do **main** and **develop**. Or, the develop branch and the main branch.

Reply:
the DevEn help hero?

Reply:
sorry @mmfred , im not sure i understand the the first point 🙈  

for the second one, i worded it to try get around putting a slack link into the doc / mentioning slack or a channel. if thats cool with yall, i  can add it in.

Reply:
You can leave as is. That's probably more consistent with our style.